mod/ckeditor/views/default/elgg/ckeditor/config.js

```diff
@@ -8,8 +8,8 @@ define(function(require) {
 		removeButtons: 'Subscript,Superscript', // To have Underline back
 		allowedContent: true,
 		baseHref: elgg.get_site_url(),
-		removePlugins: 'liststyle,contextmenu,tabletools',
-		extraPlugins: 'blockimagepaste,autogrow',
```

Comment:
Why are these removals being made? At the very least they should be mentioned in the PR description for transparency.